Makefile

```diff
@@ -270,14 +272,28 @@ kind-down: $(KIND)
 	rm -f $(REPO_ROOT)/example/provider-local/base/kubeconfig
 	rm -rf dev/local-backupbuckets
 
+
+kind-seed-up: $(KIND)
+	@if [ "$(KIND_ENV)" = "local" ]; then ip addr add 127.0.0.2 dev lo0; fi
+	$(KIND) create cluster --name seed-local --config $(REPO_ROOT)/example/gardener-local/kind/seed-cluster-$(KIND_ENV).yaml --kubeconfig $(REPO_ROOT)/example/gardener-local/kind/seed-cluster-kubeconfig
```

Comment:
nit: Should we use `seed-local2` to match the name of the seed that will be created?